<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>782</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(220, 220, 220);
selection-background-color: rgb(30, 30, 30);
selection-color: rgb(30, 30, 30);
gridline-color: rgb(30, 30, 30);
border-color: rgb(30, 30, 30);
alternate-background-color: rgb(30, 30, 30);
background-color: rgb(30, 30, 30);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>581</width>
      <height>691</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(140, 140, 140);</string>
    </property>
    <column>
     <property name="text">
      <string>Project</string>
     </property>
     <property name="background">
      <color>
       <red>255</red>
       <green>255</green>
       <blue>0</blue>
      </color>
     </property>
     <property name="foreground">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Action</string>
     </property>
     <property name="foreground">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>124</width>
      <height>56</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btnCreateProject">
       <property name="styleSheet">
        <string notr="true">color: rgb(220, 220, 220);
background-color: rgb(100 ,50,0);</string>
       </property>
       <property name="text">
        <string>Create Project</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../Icon/create.png</normaloff>../Icon/create.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRefresh">
       <property name="styleSheet">
        <string notr="true">color: rgb(220, 220, 220);
background-color: rgb(100 ,50,0);</string>
       </property>
       <property name="text">
        <string>Refresh</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../Icon/refresh.png</normaloff>../Icon/refresh.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>644</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionLaunch_Maya_2018">
   <property name="text">
    <string>Launch Maya 2018</string>
   </property>
  </action>
  <action name="actionCreateProject">
   <property name="text">
    <string>CreateProject</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
